appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -1354,8 +1353,23 @@
      ((equal? type 'key) (coerce-to-key arg))
      ((equal? type 'dictionary) (coerce-to-dictionary arg))
      ((equal? type 'any) arg)
+     ((enum-type? type) (coerce-to-enum arg type))
```

Comment:
It could be delayed, but I still have doubts about us getting that far by the end of the summer hehe. There's little harm (imo) in leaving it in, so I'd rather leave it in. Basically the same reason we left the new-style support in the ComponentProcessor.